bin/process_results.py

```diff
@@ -25,7 +25,7 @@ def truncate(text, maxlength=500):
 
 def process_results(filepath):
     output = {"version": 2, "status": "pass", "message": None, "tests": []}
-    pattern = r"(?m)^((?P<file>.*test_.*\.c):\d+:(?P<name>\w+):(?:\033\[\d+m)?(?P<status>PASS|FAIL)(?:\033\[\d+m)?(?:: (?P<message>.*))?)$"
+    pattern = r"(?m)^.*?((?P<file>test_.*\.c):\d+:(?P<name>\w+):(?:\033\[\d+m)?(?P<status>PASS|FAIL)(?:\033\[\d+m)?(?:: (?P<message>.*))?)$"
```

Comment:
I'm not really sure what this change does, can you explain @ryanplusplus ?

Reply:
Yep, basically what this does is move the `.*` prefix matcher out of the capture so that it doesn't become part of the filename. The reason the `?` is added is that it makes the match non-greedy so that it doesn't consume the filename (which is important since we want the filename capture to match the filename).